### PR TITLE
Rocket structure

### PR DESCRIPTION
**Pull request by mudasiri**
In this PR I have done the following:

1.     Created a route and a view component. Using <NavLink /> for the page navigation links and active style class to indicate which section/page the user is on (underlining the active navigation link).

2.     Create a directory for all Redux state slice files.

### CODE REVIEW

**Review by aubinleyoung — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle:

Hi @mudasiri , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has an unfilled section in your README file. Kindly fill this section to make your pull request descriptive.

Screenshot 2023-03-08 120148



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/navbar.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <div className="nav-brand">
+          <Link to="/" className="nav-img"><img src="planet.png" alt="logo" /></Link>
+          <h2>Space Travelers&apos; Hub </h2>
+        </div>
+        <ul className="nav-links">
+          <li><NavLink to="/" className={({ isActive }) => (isActive ? 'active-link nav-link' : 'nav-link')}>Rockets</NavLink></li>
+          <li><NavLink to="/categories" className={({ isActive }) => (isActive ? 'active-link nav-link' : 'nav-link')}>Missions</NavLink></li>
+          <span>|</span>
+          <li><NavLink to="/categories" className={({ isActive }) => (isActive ? 'active-link nav-link' : 'nav-link')}>My Profile</NavLink></li>
+        </ul>
```
Comment (aubinleyoung) [inline src/components/navbar.js:17]: - [ ] As shown in the image below the Rockets link is underlined when it is active but Missions and My profile links are active and underlined at the same time, kindly fix this by separating those two active links as required to indicate which section/page the user is currently on. 
Screenshot 2023-03-08 135720

Screenshot 2023-03-08

**Review by aubinleyoung — CHANGES_REQUESTED** [review]
## Hi @mudasiri ,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

 

- [ ] You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [ ] I can see in your project there is no directory for all Redux state slice files, kindly add it as required and as mentioned in the description of your PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/navbar.js (hunk 1 above)
Comment (mudasiri) [reply src/components/navbar.js:17]: Thanks for your comment. As i taught this was required in the other task so I have fixed that in the fetch PR

**Review by aubinleyoung — APPROVED** [review]
# STATUS: MILESTONE APPROVED

Hi @mudasiri  , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  All requirements completed
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉

**Pull request merged**